lms/templates/admin/macros.jinja

```diff
@@ -64,55 +64,66 @@
 {% endmacro %}
 
 {% macro instances_table(request, instances) %}
-<table class="table is-fullwidth">
-  <thead>
-    <tr>
-      <th>ID</th>
-      <th>Consumer Key</th>
-      <th>GUID</th>
-      <th>Issuer</th>
-      <th>Deployment ID</th>
-      <th></th>
-    </tr>
-  </thead>
-  <tbody>
-{% for instance in instances %}
-    <tr>
-      <td>{{instance.id}}</td>
-      <td>{{instance.consumer_key}}</th>
-      <td>{{instance.tool_consumer_instance_guid}}</td>
-      <td>{{instance.lti_registration.issuer}}</td>
-      <td>{{instance.deployment_id}}</td>
-      <td>
-        <a class="button" href="{{ request.route_url("admin.instance.id", id_=instance.id) }}">View</a>
-      </td>
-    </tr>
-{% endfor %}
+<div class="container">
```

Comment:
This two `div` allow for the overflow on the table:

https://bulma.io/documentation/elements/table/#table-container